src/shareon.css

```diff
@@ -207,3 +207,10 @@
 .shareon > .whatsapp:before {
   background-image: url("icons/whatsapp.svg");
 }
+
+.shareon > .hackernews {
+  background-color: #ff6600;
```

Comment:
**issue (non-blocking):** This is the colour of the Hacker News' header, but not the official colour used by Y Combinator. I'd prefer the latter.

**suggestion:** Use the colour from the Y Combinator logo: `#fb651e` 

```suggestion
  background-color: #fb651e;
```